src/main/java/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,28 @@
+package cinema.controller;
+
+import cinema.dto.mapper.impl.UserMapper;
+import cinema.dto.request.UserRequestDto;
+import cinema.exception.RegistrationException;
+import cinema.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
+public class AuthenticationController {
+    private final UserMapper userMapper;
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(UserMapper userMapper,
+                                    AuthenticationService authenticationService) {
+        this.userMapper = userMapper;
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserRequestDto dto) throws RegistrationException {
```

Comment:
```suggestion
    public void register(@RequestBody UserRequestDto dto) {
```
I didn't see you throw this anywhere